packages/playground/website/.htaccess

```diff
@@ -5,7 +5,7 @@ AddEncoding x-gzip .gz
 Header unset ETag
 Header set Cache-Control "max-age=0, no-cache, no-store, must-revalidate"
 </FilesMatch>
-<FilesMatch "index\.js|blueprint-schema\.json|logger.php|wp-cli.phar|wordpress-importer.zip">
+<FilesMatch "index\.js|blueprint-schema\.json|logger.php|puzzle.php|wp-cli.phar|wordpress-importer.zip">
```

Comment:
@bgrgicak for migrating to the new host, this change needs a corresponding change to the list here:
https://github.com/WordPress/wordpress-playground/pull/1372/files#diff-5130b375ec48ad9f9bcccc2abcf2f8551850f7667d6e1c1359de6e22d12f973eL293-L300

Reply:
Thanks, I added it now [485d932](https://github.com/WordPress/wordpress-playground/pull/1372/commits/485d932bb77d77c4445680bee81c862c34f1d6a6)

Reply:
> Thanks, I added it now 485d932

Thank you!